frontend/src/pages/404.js

```diff
@@ -1,7 +1,21 @@
-import { Typography } from "@mui/material";
+import { Button, Typography } from "@mui/material";
 
 const NotFound = () => {
-  return <Typography variant="h3">404 - Page Not Found</Typography>;
+  return (
+    <div>
```

Comment:
Can you also center the whole thing? And maybe put it in a [Card Component](https://mui.com/material-ui/react-card/)?